app/controllers/admin/edition_tags_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Admin::EditionTagsController < Admin::BaseController
+  before_filter :find_edition
+  before_filter :enforce_permissions!
+  before_filter :limit_edition_access!
+
+  def enforce_permissions!
```

Comment:
It's best to make this method private (Rails does magic things to public methods on controllers).